app/views/declare-partnership/multiple/failed.html

```diff
@@ -0,0 +1,73 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+    Upload schools
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">
+                Upload schools
+            </h1>
+            {{ govukWarningText({
+                text: "You should only add a school to your list if they have signed a partnership contract with you",
+                iconFallbackText: "Warning"
+            }) }}
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                labore
+                et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut
+                aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit
+                esse
+                cillum dolore eu fugiat nulla pariatur.
+            </p>
+            <p>
+                Please ensure that you CSV is complete with th following data fields
+            </p>
+            <ul>
+                <li>school URN</li>
+                <li>school name</li>
+                <li>school address</li>
+                <li>school point of contact name</li>
+                <li>school point of contact email address</li>
+                <li>estimated number of ECTs</li>
+                <li>assigned delivery partner</li>
+            </ul>
+            <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
+                 data-module="govuk-error-summary">
```

Comment:
What does `tabindex="-1"` do?

Reply:
Taken from https://design-system.service.gov.uk/components/error-summary/
Not sure exactly why they don't want to make it tabable...

Reply:
I guess so that when you tab through the form with validation errors it will be more explicitly noticeable. Alright, makes sense. 

Is there a reason we use the html snippet rather than nunjucks one? Not that it matters for this PR.

Reply:
Because I wanted to add a bulletpoint list. I think I could have done with nunjucks, but it was easier this way